completions/rpm

```diff
@@ -244,30 +257,32 @@ _comp_cmd_rpmbuild()
     local cur prev words cword was_split comp_args
     _comp_initialize -s -- "$@" || return
 
-    local rpm="${1%build*}"
-    [[ $rpm == "$1" ]] || ! type "$rpm" &>/dev/null && rpm=
-
     local noargopts='!(-*|*[rED]*)'
-    # shellcheck disable=SC2254
+    # shellcheck disable=SC2119,SC2254
     case $prev in
         --buildroot | --root | --dbpath | -${noargopts}r)
             _comp_compgen filedir -d
             return
             ;;
+        --target | --eval | -${noargopts}E | --buildpolicy)
+            local pathcmd
+            pathcmd=$(type -P "$1") && local PATH=${pathcmd%/*}:$PATH
```

Comment:
The benefit of prepending to `$PATH` as opposed to trying to figure out the exact command is that we don't need to poke into the internals of the utility function even for file-local use, and the utility function's default command basename persists.

In some cases we may want to pass the full path, that's still possible, but I suppose the `$PATH` modification use case would be a more common one.